src/sonic-framework/rebootbackend/reboot_common.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include <time.h>
+
+#include "status_code_util.h"
+
+namespace rebootbackend {
+
+//extern bool sigterm_requested;
```

Comment:
Let's remove this. I know some sonic code has commented line but that usually ended up forgotten and become tech debts. If there is no immediate usage let's just remove it. We can always add them back.

Reply:
Pl. refer to the latest PR https://github.com/sonic-net/sonic-buildimage/pull/20713. I have removed the comment, since this is required by tests folder.